src/cpu/callback.cpp

```diff
@@ -543,17 +569,19 @@ callback_number_t CALLBACK_SetupExtra(callback_number_t cb_num, Bitu type, PhysP
 			phys_writeb(physAddress+0x1B,(uint8_t)0xC3);	//A RETN Instruction
 		return (use_cb?32:27);
 	default:
-		E_Exit("CALLBACK:Setup:Illegal type %" sBitfs(u),type);
+		E_Exit("Unknown callback type 0x%04x", callback_type);
```

Comment:
Would it be better to just log an error and return without hard exiting? I really dislike hard exits.

Reply:
OK, changed - no more `E_Exit` in the reworked function.